kong/plugins/jwt/asn_sequence.lua

```diff
@@ -0,0 +1,82 @@
+local error = error
+
+local _M = {}
+_M.__index = _M
+
+function _M.create_simple_sequence(input)
+  if type(input) ~= "table" then error("Argument #1 must be a table", 2) end
+  local sortTable = {}
+  for pair in pairs(input) do
+    table.insert(sortTable, pair)
+  end
+  table.sort(sortTable)
+  local numbers = ""
+  for i,n in ipairs(sortTable) do
+    if type(n) ~= "number" then error("Table must use numbers as keys", 2) end
+    local number = input[sortTable[i]]
+    if type(number) ~= "string" then error("Table contains non-string value.", 2) end
+    local length = #number
+    if length > 0x7F then error("Mult-byte lengths are not supported") end
+    numbers = numbers .. "\x02" .. string.char(length) .. number
+  end
+  if #numbers > 0x7F then error("Multi-byte lengths are not supported") end
+  return "\x30" .. string.char(#numbers) .. numbers
+end
+
+function _M.parse_simple_sequence(input)
+  if type(input) ~= "string" then error("Argument #1 must be string", 2) end
+  if #input == 0 then error("Argument #1 must not be empty", 2) end
+  if string.byte(input, 1) ~= 0x30 then error("Argument #1 is not a sequence") end
+  local length = string.byte(input, 2)
+  if length == nil then error("Sequence is incomplete")
+  elseif length > 0x7F then error("Multi-byte lengths are not supported")
+  elseif length ~= #input-2 then error("Sequence's asn length does not match expected length") end
+  local seq = {}
+  local counter = 1
+  local position = 3
+  while true do
+    if position == #input+1 then break
+    elseif position > #input+1 then error("Sequence moved out of bounds.")
+    elseif counter > 0xFF then error("Sequence is too long") end
+    local chunk = string.sub(input, position)
+    if string.byte(chunk, 1) ~= 0x2 then error("Sequence did not contain integers") end
```

Comment:
Code formatting; please stick to the formatting used in the rest of the codebase.

eg.
```lua
if position == #input+1 then 
  break
elseif position > #input+1 then 
  error("Sequence moved out of bounds.")
elseif counter > 0xFF then 
  error("Sequence is too long") 
end
```